cmd/jaeger/internal/extension/jaegerstorage/config.go

```diff
@@ -13,9 +14,16 @@ type Config struct {
 	// TODO add other storage types here
 	// TODO how will this work with 3rd party storage implementations?
 	//      Option: instead of looking for specific name, check interface.
+
+	ClickHouse map[string]ch.Config `mapstructure:"clickhouse"`
 }
 
 type MemoryStorage struct {
 	Name string `mapstructure:"name"`
 	memoryCfg.Configuration
 }
+
+type ClickHouseStorage struct {
```

Comment:
is this type used?

Reply:
Ah, I followed your MemoryStorage struct above. Now realize I'm currently not needing this for anything. Do you use MemoryStorage struct for anything?

Reply:
not needed, you can remove both